force-app/main/default/lwc/nksSurvey/nksSurvey.js

```diff
@@ -0,0 +1,111 @@
+import { LightningElement, wire } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import getSurvey from '@salesforce/apex/NKS_InternalSurveyController.getSurvey';
+import hasAnswered from '@salesforce/apex/NKS_InternalSurveyController.hasAnswered';
+import createAnsweredRecord from '@salesforce/apex/NKS_InternalSurveyController.createAnsweredRecord';
+import createResponseRecord from '@salesforce/apex/NKS_InternalSurveyController.createResponseRecord';
+export default class NksSurvey extends LightningElement {
+    surveyId;
+    backgroundColor;
+    title;
+    question;
+    isAnswered;
+    isRendered = false;
+    startDate;
+    endDate;
+    rating;
+    comment;
+    show = true;
```

Comment:
Denne burde være default false, eller så popper surveyen opp i et sekund eller to når veilederen har svart.